### PR TITLE
Add OSTree based deployments

### PR DESCRIPTION
**Pull request by oglok**
Signed-off-by: Ricardo Noriega <rnoriega@redhat.com>

Closes [#631](https://github.com/redhat-et/microshift/issues/631)

### CODE REVIEW

File: content/en/docs/getting-started/_index.md [hunk 1]
```diff
@@ -220,3 +220,29 @@ oc.exe get all -A --kubeconfig .\kubeconfig
...
+```
+curl -L -o /etc/yum.repos.d/fedora-modular.repo https://src.fedoraproject.org/rpms/fedora-repos/raw/rawhide/f/fedora-modular.repo
+curl -L -o /etc/yum.repos.d/fedora-updates-modular.repo https://src.fedoraproject.org/rpms/fedora-repos/raw/rawhide/f/fedora-updates-modular.repo
+curl -L -o /etc/yum.repos.d/group_redhat-et-microshift-fedora-35.repo https://copr.fedorainfracloud.org/coprs/g/redhat-et/microshift/repo/fedora-35/group_redhat-et-microshift-fedora-35.repo
+rpm-ostree ex module enable cri-o:1.21
+
+rpm-ostree upgrade
+systemctl reboot
+
+rpm-ostree install cri-o cri-tools microshift
```
Comment (mangelajo) [inline content/en/docs/getting-started/_index.md:246]:
```suggestion
rpm-ostree install cri-o cri-tools microshift
systemctl reboot
```
? I havent tried, but wondering if we need it.

Comment (oglok) [reply content/en/docs/getting-started/_index.md:246]: Do you mean if we need 2 reboots?

Comment (oglok) [reply content/en/docs/getting-started/_index.md:244]: we only need one reboot. You can upgrade + install microshift without an intermidiate reboot. Thanks for the catch!

**Review by mangelajo — APPROVED** [review]
/lgtm

**Pull request merged**